pkg/repositories/config/migrations.go

```diff
@@ -330,18 +330,62 @@ var Migrations = []*gormigrate.Migration{
 		},
 	},
 
-	// Add state to execution model. For any new table migrations please user the following pattern due to following bug
+	// For any new table, Please use the following pattern due to a bug
 	// in the postgres gorm layer https://github.com/go-gorm/postgres/issues/65
 	{
-		ID: "2022-01-11-execution-state",
+		ID: "2022-01-11-id-to-bigint",
```

Comment:
can we make this a new, separate migration and add a rollback just in case users face issues?

Reply:
Done. its part of two different migration now and also added rollback.